tests/components/onboarding/test_views.py

```diff
@@ -29,6 +30,57 @@ def auth_active(hass):
     )
 
 
+@pytest.fixture(name="rpi")
+async def rpi_fixture(hass, aioclient_mock, mock_supervisor):
+    """Mock core info with rpi."""
+    aioclient_mock.get(
+        "http://127.0.0.1/core/info",
+        json={
+            "result": "ok",
+            "data": {"version_latest": "1.0.0", "machine": "raspberrypi3"},
+        },
+    )
+    assert await async_setup_component(hass, "hassio", {})
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6243615)